src/locale/locales/fr/messages.po

```diff
@@ -4360,21 +4175,17 @@ msgstr "Utiliser le fournisseur par défaut"
 #: src/view/com/modals/InAppBrowserConsent.tsx:56
 #: src/view/com/modals/InAppBrowserConsent.tsx:58
 msgid "Use in-app browser"
-msgstr ""
+msgstr "Utiliser le navigateur dans l’appli"
```

Comment:
`in-app` is not always easy to translate, but here I think that `interne` is the best to convey the meaning.

```suggestion
msgstr "Utiliser le navigateur interne à l’appli"
```